code/style.css

```diff
@@ -22,37 +53,66 @@ p {
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: #E7B4C2;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
 }
 
+.killer-deck:hover {
+  transform: scale(1.1);
```

Comment:
I like this transform you have created! Have you played around with `transition` in css? That will give you a nice transition :D